playbooks/common/openshift-cluster/upgrades/v3_3/master_config_upgrade.yml

```diff
@@ -48,3 +48,7 @@
     dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
     yaml_key: 'controllerConfig.servicesServingCert.signer.keyFile'
     yaml_value: service-signer.key
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'admissionConfig.pluginConfig'
+    yaml_value: "{{ openshift.master.kube_admission_plugin_config | to_padded_yaml(level=2) }}"
```

Comment:
Is this for 3.4? If so you'll need to add a "hook" file for 3.4 similar to this one, and link it similar to how this one is for the upgrade process. (just grep for master_config_upgrade)